tests/mr/host/pinned_pool_mr_tests.cpp

```diff
@@ -33,7 +32,7 @@ using pool_mr = rmm::mr::pool_memory_resource<rmm::mr::pinned_memory_resource>;
 
 TEST(PinnedPoolTest, ThrowOnNullUpstream)
 {
-  auto construct_nullptr = []() { pool_mr mr{nullptr}; };
+  auto construct_nullptr = []() { pool_mr mr{nullptr, 0}; };
```

Comment:
is zero a valid pool size? If not, we should use a valid size to make sure the throw is because of the nullptr.

Reply:
Fixed anyway. Thanks!